### ISSUE TITLE
NTR: MRI: superficial siderosis (Lab: radiology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8895]
New term request MRI: superficial siderosis (jadhav_091113160351): Linear gyriform pattern of hypointense signal on T2-weighted GRE  UMLS: 2938918

**Comment by pnrobinson** [comment]
@MickeySegal
it seems there are two types of superficial siderosis, see https://radiopaedia.org/articles/superficial-siderosis-of-the-central-nervous-system

What is the medical context of your finding? We should probably create two terms.

**Comment by MickeySegal** [comment]
We use this only for "Superficial siderosis of the central nervous system": https://pubmed.ncbi.nlm.nih.gov/7655881/

From the Radiopaedia reference, "Cortical superficial siderosis" sounds like something separate.

**Comment by pnrobinson** [comment]
@MickeySegal 
I think that Superficial siderosis is best characterized as a disease. I think it would be better to have terms for the MRI phenotype. I would suggest the following

**Superficial CNS hemosiderin deposition** 
def: An anomaly that can be ascertained by iron-specific magnetic resonance imaging (MRI) sequences such as gradient-echo (GRE) and susceptibility-weighted imaging (SWI). Superficial siderosis of the central nervous system is a chronic condition due to persistent or repeated long-term bleeding into the subarachnoid space, leading to subpial hemosiderin deposition. Hemosiderin deposition on MRI is seen as a T2 hypointensity and preferentially outlines the cerebellum and brainstem surface. PMID: [38753899](https://pubmed.ncbi.nlm.nih.gov/38753899).
RELATED SYNONYM: Superﬁcial siderosis
Comment: Superficial CNS hemosiderin deposition is a manifestation of superﬁcial siderosis (SS) of the central nervous system (CNS), which is typically due to chronic low-grade extravasation of blood into the subarachnoid space. Hemoglobin in red blood cells in the cerebrospinal ﬂuid (CSF) breaks down into globin and neurotoxic heme. In response to the presence of neurotoxic heme, neuroglial cells release hemoxygenase and apoferritin. Hemoxygenase breaks heme into free-iron and biliverdin, and apoferritin binds free-iron to form ferritin and subsequently hemosiderin. This process is likely neuroprotective and prevents free-iron mediated neuronal injury. Persistence of blood in the subarachnoid space eventually overwhelms the ferritin production capacity and unbound iron results in free-radical mediated neuronal injury. Hemosiderin deposits in regions of the brain or spinal cord adjacent to CSF. This conﬂuent subpial hemosiderin deposition results in a characteristic marginal and conﬂuent T2-hypointensity on magnetic resonance imaging (MRI) that is referred to as SS and it is best appreciated on iron-sensitive sequences.
PARENT: Abnormal meningeal morphology HP:0010651. (is there a better parent?)
PMID:38753899
PMID:33860558

Children
**Cortical superficial CNS hemosiderin deposition** 
def: A type of superficial CNS hemosiderin deposition that most commonly occurs due to iron deposits in the superficial cortical layers following bleeding from leptomeningeal vessels, most often from cerebral amyloid angiopathy in patients aged 55 or older.
RELATED SYNONYM: Cortical superﬁcial siderosis
PMID:38753899
PMID:33860558

**Infratentorial superficial CNS hemosiderin deposition**
def: A type of superficial CNS hemosiderin deposition that involves the cerebellum and brainstem, most commonly the superior vermis, quadrigeminal plate, and basal cerebral surfaces. Cerebellar atrophy, particularly the superior vermis and anterior cerebellar hemispheres, can also be seen on a computed tomography (CT) scan of the head. The MRI of the spinal cord may reveal myelomalacia, syringomyelia, “owl-eyes,” or “snake-eyes” signs. The most common etiology for classic or primary infratentorial superficial siderosis (iSS) is spinal dural defects, often dural tears. 
RELATED SYNONYM: Infratentorial Superficial Siderosis
PMID:38753899
PMID:33860558


Thoughts?

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001255
- label: Superficial CNS hemosiderin deposition
- parents: [Abnormal meningeal morphology (HP:0010651)](https://hpo.jax.org/app/browse/term/HP:0010651)
- pmid: PMID:38753899
- def: An anomaly that can be ascertained by iron-specific magnetic resonance imaging (MRI) sequences such as gradient-echo (GRE) and susceptibility-weighted imaging (SWI). Superficial siderosis of the central nervous system is a chronic condition due to persistent or repeated long-term bleeding into the subarachnoid space, leading to subpial hemosiderin deposition. Hemosiderin deposition on MRI is seen as a T2 hypointensity and preferentially outlines the cerebellum and brainstem surface.
- comment: Superficial CNS hemosiderin deposition is a manifestation of superficial siderosis (SS) of the central nervous system (CNS), which is typically due to chronic low-grade extravasation of blood into the subarachnoid space. Hemoglobin in red blood cells in the cerebrospinal fluid (CSF) breaks down into globin and neurotoxic heme. In response to the presence of neurotoxic heme, neuroglial cells release hemoxygenase and apoferritin. Hemoxygenase breaks heme into free iron and biliverdin, and apoferritin binds free iron to form ferritin and subsequently hemosiderin. This process is likely neuroprotective and prevents free iron mediated neuronal injury. Persistence of blood in the subarachnoid space eventually overwhelms the ferritin production capacity and unbound iron results in free-radical mediated neuronal injury. Hemosiderin deposits in regions of the brain or spinal cord adjacent to CSF. This confluent subpial hemosiderin deposition results in a characteristic marginal and confluent T2-hypointensity on magnetic resonance imaging that is referred to as SS and it is best appreciated on iron-sensitive sequences.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8895
- synonyms: Superﬁcial siderosis(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001256
- label: Cortical superficial CNS hemosiderin deposition
- parents: [Abnormal meningeal morphology (HP:0010651)](https://hpo.jax.org/app/browse/term/HP:0010651)
- pmid: PMID:38753899|PMID:33860558
- def: A type of superficial CNS hemosiderin deposition that most commonly occurs due to iron deposits in the superficial cortical layers following bleeding from leptomeningeal vessels, most often from cerebral amyloid angiopathy in patients aged 55 or older.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8895
- synonyms: Cortical superﬁcial siderosis(NONE)

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001257
- label: Infratentorial superficial CNS hemosiderin deposition
- parents: [Abnormal meningeal morphology (HP:0010651)](https://hpo.jax.org/app/browse/term/HP:0010651)
- pmid: PMID:38753899|PMID:33860558
- def: A type of superficial CNS hemosiderin deposition that involves the cerebellum and brainstem, most commonly the superior vermis, quadrigeminal plate, and basal cerebral surfaces. Cerebellar atrophy, particularly the superior vermis and anterior cerebellar hemispheres, can also be seen on a computed tomography scan of the head. The MRI of the spinal cord may reveal myelomalacia, syringomyelia, owl-eyes, or snake-eyes signs. The most common etiology for classic or primary infratentorial superficial siderosis is spinal dural defects, often dural tears.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8895
- synonyms: Infratentorial Superficial Siderosis(NONE)
